resources/dw/cluster/aws/resource_cluster.go

```diff
@@ -100,6 +105,29 @@ func (r *dwClusterResource) Create(ctx context.Context, req resource.CreateReque
 		return
 	}
 
+	if plan.PollingOptions.Async.ValueBool() {
+		callFailedCount := 0
+		stateConf := &retry.StateChangeConf{
+			Pending:      []string{"Creating", "Created", "Setting Up", "Accepted", "Starting"},
```

Comment:
IMO there is not such state `Setting Up`

Reply:
It should be 
```
          - Creating
          - Created
          - Accepted
          - Starting
          - Running
          - Stopping
          - Stopped
          - Updating
          - PreUpdate
          - PreUpgrade
          - Restarting
          - Deleting
          - Upgrading
          - Waiting
          - Error
          - Failed
```